pkg/devcontainer/prebuild.go

```diff
@@ -15,6 +15,10 @@ func (r *Runner) Build(ctx context.Context, options config.BuildOptions) (string
 		return "", err
 	}
 
+	if options.Repository == "" && len(config.GetDevPodCustomizations(substitutedConfig.Config).PrebuildRepository) == 0 {
```

Comment:
Is it possible to have this check in the PreRun part of the build command like in my PR #602 . Wdyt?